Formula/openliberty-jakartaee8.rb

```diff
@@ -0,0 +1,46 @@
+class OpenlibertyJakartaee8 < Formula
+  desc "Lightweight open framework for Java (Jakarta EE 8)"
+  homepage "https://openliberty.io"
+  url "https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/2021-06-29_1900/openliberty-javaee8-21.0.0.7.zip"
+  sha256 "fe8db0f5ef34aebc81b0af8e3e511672b02c9efe51f0a40c3675b28752d280be"
+  license "EPL-1.0"
+
+  livecheck do
+    url "https://openliberty.io/api/builds/data"
+    regex(/openliberty[._-]v?(\d+(?:\.\d+)+)\.zip/i)
+  end
+
+  depends_on "openjdk"
+
+  def install
+    rm_rf Dir["bin/**/*.bat"]
+
+    prefix.install %w[LICENSE NOTICES README.TXT]
+    libexec.install Dir["*"]
+    (bin/"openliberty-jakartaee8").write_env_script "#{libexec}/bin/server",
+                                                    Language::Java.overridable_java_home_env
+  end
+
+  def caveats
+    <<~EOS
+      The home of Open Liberty Jakarta EE 8 is:
+        #{opt_libexec}
+      To run Open Liberty Jakarta EE 8:
+        #{opt_libexec}/bin/openliberty-jakartaee8 start
```

Comment:
It's in path because you're installing it to `bin/"openliberty-jakartaee8"`
```suggestion
      To run Open Liberty Jakarta EE 8:
        openliberty-jakartaee8 start
```